Samples/kotlin/src/main/res/layout/activity_routing.xml

```diff
@@ -1,165 +1,186 @@
-            android:paddingEnd="5dp"
-            android:textColor="@color/outer_space"
-            android:textSize="14sp"
-            tools:text="Arrival Time 9:52 PM" />
-
-        <Button
-            android:id="@+id/button_exit_route"
-            android:layout_width="144dp"
+            android:layout_above="@+id/fab"
+            android:layout_alignParentEnd="true"
+            android:layout_marginBottom="17dp"
+            android:background="@color/semi_transparent"
+            android:orientation="vertical"
+            android:padding="6dp"
+            app:layout_constraintBottom_toTopOf="@+id/fab"
+            app:layout_constraintEnd_toEndOf="@+id/fab">
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/switch_floors" />
+
+            <Spinner
+                android:id="@+id/floorSpinner"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content" />
+
+        </LinearLayout>
+
+        <android.support.design.widget.FloatingActionButton
+            android:id="@+id/fab"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_alignParentEnd="true"
-            android:background="@color/exit_red"
-            android:fontFamily="sans-serif-medium"
-            android:text="@string/demo_walk_time_exit"
-            android:textAllCaps="false"
-            android:textColor="@color/white"
-            android:textSize="18sp" />
-
-    </RelativeLayout>
-
-</android.support.constraint.ConstraintLayout>
\ No newline at end of file
+            android:layout_alignParentBottom="true"
+            android:layout_gravity="bottom|end"
+            android:layout_marginLeft="16dp"
+            android:layout_marginEnd="16dp"
+            android:layout_marginRight="16dp"
+            android:layout_marginBottom="16dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:srcCompat="@drawable/ic_navigation" />
+
+        <fragment
+            android:id="@+id/routeSummaryFragment"
+            android:name="com.phunware.kotlin.sample.RouteSummaryFragment"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toTopOf="@+id/walk_time_view"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <android.support.constraint.ConstraintLayout
+            android:id="@+id/walk_time_view"
+            android:layout_width="match_parent"
+            android:layout_height="110dp"
+            android:background="@color/white"
+            android:elevation="12dp"
+            android:outlineProvider="paddedBounds"
```

Comment:
I can't recall from your screen share if "paddedBounds" had any effect over not providing this attribute at all.  If that's the case, can we remove the attribute?

Reply:
darnit that was left in on accident

Reply:
actually nick i think it does work. paddedBound does give us the shadow

Reply:
nvm you were right.